CMSgov/onemac
# Oy2 design changes

Story: https://qmacbis.atlassian.net/browse/OY2-9105
Endpoint: https://d2e8275h705g6u.cloudfront.net/

### Details

Design changes on the FAQ, User Profile, and Phone Number edit features

### Changes

Style-based design changes.

### Test Plan
Test the following changes are implemented as in the description and Figma page.

 Check if the following issues are fixed:

FAQ content Update
===============
1. The gradient bar being is off in Helpdesk box
2. The Headings are out of sequence (H2 and H3 are mixed up)
![image](https://user-images.githubusercontent.com/12236683/122325415-8c8a0980-cef8-11eb-982f-bf01552a1c74.png)
3. Padding is not the same for all expanded questions on the FAQ page. The padding should be 24px between question and answer.
![image](https://user-images.githubusercontent.com/12236683/122325476-aa576e80-cef8-11eb-9d1e-d6f69699b61d.png)
4. Indent is off when expanding links
5. The Plus and Minus icons must use SVG files instead of PNG
6. Make sure the Helpdesk contact box doesn't get too close to the main FAQ contents when the screen size is reduced
7. Make sure the Helpdesk contact box is of 100% width of the container on a smaller screens

User Profile: State User Requests additional state access
========================================
1. “select state here” should probably be changed to sentence case (ex: “Select state here”)
2. “Submission banner” change content, confusing with “Submission Success” as the title for all alerts

Users can edit phone number
=====================
1. Green save button change to blue.
